examples/Example.SmartContract.NEP11/SampleNep11Token.cs

```diff
@@ -0,0 +1,208 @@
+        public static event OnSetMinterDelegate OnSetMinter;
+
+        public static void SetMinter(UInt160 newMinter)
+        {
+            if (IsOwner() == false)
+                throw new InvalidOperationException("No Authorization!");
+            if (!newMinter.IsValid) return;
+            Storage.Put(new[] { PrefixMinter }, newMinter);
+            OnSetMinter(newMinter);
+        }
+
+        public static void Mint(UInt160 to)
+        {
+            if (IsOwner() == false && IsMinter() == false)
+                throw new InvalidOperationException("No Authorization!");
+            IncreaseCount();
+            BigInteger tokenId = CurrentCount();
+            Nep11TokenState nep11TokenState = new Nep11TokenState()
+            {
+                Name = "SampleNep11Token",
+                Owner = to
+            };
+            Mint((ByteString)tokenId, nep11TokenState);
+        }
+
+        private static void SetCount(BigInteger count)
+        {
+            StorageMap map = new(Storage.CurrentContext, PrefixCount);
+            map.Put("counter", count);
+        }
+
+        [Safe]
+        public static BigInteger CurrentCount()
+        {
+            StorageMap map = new(Storage.CurrentReadOnlyContext, PrefixCount);
+            var data = map.Get("counter");
+            if (data is not null)
+            {
+                return (BigInteger)data;
+            }
+            else
+            {
+                return 0;
+            }
+        }
+
+        private static void IncreaseCount()
+        {
+            SetCount(CurrentCount() + 1);
+        }
+
+        #endregion
+
+        #region Example.SmartContract.NEP11
+
+        public override string Symbol { [Safe] get => "SampleNep11Token"; }
+
+        #endregion
+
+        #region Royalty
+        private const byte PrefixRoyalty = 0xfb;
+
+        private static readonly UInt160 InitialRecipient = "NUuJw4C4XJFzxAvSZnFTfsNoWZytmQKXQP";
+        private static readonly BigInteger InitialFactor = 700;
+
+        internal static void SetRoyaltyInfo(ByteString tokenId, Map<string, object>[] royaltyInfos)
+        {
+            if (IsOwner() == false)
+                throw new InvalidOperationException("No Authorization!");
+            for(uint i = 0; i < royaltyInfos.Length; i++)
+            {
+                if (((UInt160)royaltyInfos[i]["royaltyRecipient"]).IsValid == false ||
+                    (BigInteger)royaltyInfos[i]["royaltyRecipient"] < 0 ||
+                    (BigInteger)royaltyInfos[i]["royaltyRecipient"] > 10000
+                    )
+                    throw new InvalidOperationException("Parameter error");
+            }
+            StorageMap map = new(Storage.CurrentContext, PrefixRoyalty);
+            map.Put(tokenId, StdLib.Serialize(royaltyInfos));
+        }
+
+        [Safe]
+        internal static Map<string, object>[] RoyaltyInfo(ByteString tokenId, UInt160 royaltyToken, BigInteger salePrice)
```

Comment:
```suggestion
        public static Map<string, object>[] RoyaltyInfo(ByteString tokenId, UInt160 royaltyToken, BigInteger salePrice)
```
Figure out if you want this function to be exposed in the `abi` or not? `public` or `private`